Plugins/Effect/NWScript/nwnx_effect.nss

```diff
@@ -354,3 +360,11 @@ int NWNX_Effect_RemoveEffectById(object oObject,  string sID)
 
     return  NWNX_GetReturnValueInt();
 }
+
+void ApplyEffect(effect eEffect, object oObject)
+{
+    string sFunc = "ApplyEffect";
+    NWNX_PushArgumentEffect(NWNX_Effect, sFunc, eEffect);
+    NWNX_PushArgumentObject(NWNX_Effect, sFunc, oObject);
```

Comment:
The arguments for NWNX_{Push|Pop}Argument{Type} have changed, they no longer take the Plugin/Function name.

Eg:

   NWNX_PushArgumentEffect(eEffect);
    NWNX_PushArgumentObject(oObject);

Don't forget to update your own nwnx.nss and other nwnx_*.nss scripts!